client/pages/index.tsx

```diff
@@ -27,44 +27,69 @@ const Home = () => {
   const { takePhoto } = useCamera();
 
   const [pageState, setPageState] = useState(pageStates.MAIN);
+  const [productInfo, setProductInfo] = useState<ProductData[]>([]);
 
   const handleStart = () => {
-    console.log("Starting countdown...");
+    console.log('Starting countdown...');
     setPageState(pageStates.COUNTDOWN);
   };
 
   const closeCarousel = () => {
-    console.log("Closing carousel and returning to main state.");
+    console.log('Closing carousel and returning to main state.');
     setPageState(pageStates.MAIN);
   };
 
   const onCountdownComplete = () => {
-    console.log("Countdown complete. Moving to SMILE state...");
+    console.log('Countdown complete. Moving to SMILE state...');
     setPageState(pageStates.SMILE);
   };
 
   const takePhotoHandler = async () => {
-    console.log("Attempting to capture photo...");
+    console.log('Attempting to capture photo...');
     try {
       const formData = await takePhoto();
       if (formData) {
         console.log('Photo captured successfully.');
         console.log('FormData contains image:', formData.get('image'));
-
-        await saveImage(formData);
       } else {
         console.log('No FormData generated by takePhoto.');
+        // TODO: add an error message state
+      }
+
+      // ********************************************************************
+      // NOTE: Nov 17
```

Comment:
instead of calling the `fetch` request directly in the code, let's move it to a service (use CameraService as a reference -- content-type should also be formdata) 
and use `axios` instead of fetch